enqueue/fs/0.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/",
+        "var/": "%VAR_DIR%"
+    },
+    "env": {
+        "ENQUEUE_FS_STORAGE": "%VAR%/queue"
+    },
+    "gitignore": [
+        "/var/queue"
```

Comment:
Yeah, I know. It is done in framework bundle and what if someone did not install it. That is here just for that case.